config/locales/idv/es.yml

```
@@ -297,6 +297,7 @@ es:
       review: Revise y envíe
       session:
         review: 'Vuelve a ingresar tu contraseña de %{app_name}'
+        review_letter: 'Vuelve a ingresar tu contraseña de %{app_name} para enviar su carta'
```

This has informal and formal you in the same sentence.
```suggestion
        review_letter: 'Vuelve a ingresar su contraseña de %{app_name} para enviar su carta'
```